fjord/feedback/views.py

```diff
@@ -135,10 +142,10 @@ def _handle_feedback_post(request, locale=None, product=None,
     data = form.cleaned_data
 
     description = data.get('description', u'').strip()
+
     if not description:
-        # If there's no description, then there's nothing to do here,
-        # so thank the user and move on.
-        return HttpResponseRedirect(reverse('thanks'))
+        # If there's no description populate a generic one
+        description = u'Sentiment: %s' % (('Sad', 'Happy')[data['happy']])
```

Comment:
Input feedback isn't a rating system--it's a system for gathering textual sentiment which requires descriptions. That's why if there's no description, we just drop it and move on.

Do you have a requirement for feedback with no descriptions? If so, that'll require other changes including to the ratelimiting which will likely have some big ramifications.